rpc_jobs/periodic_cleanup.yml

```diff
@@ -0,0 +1,57 @@
+- job:
+    name: 'Periodic-Cleanup'
+    project-type: workflow
+    parameters:
+      - string:
+          name: "INSTANCE_AGE_LIMIT"
+          default: "48"
+          description: |
+            Hours. Instances older than this will be removed.
+      - string:
+          name: "INSTANCE_PREFIX"
+          default: "ra"
+          description: |
+            Only instances whose names match the supplied prefix will be cleaned
+            up.
+      - string:
+          name: "OS_REGION_NAME"
```

Comment:
Good point, I was using OS_REGION_NAME because it matched the other vars used by nova client, but this PR no longer uses nova client, so using REGION to be consistent with the rest of the repo would be better.